data/core/units/orcs/Adept.cfg

```diff
@@ -0,0 +1,99 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Orcish Adept
+    name= _ "Orcish Adept"
+    race=orc
+    image="units/orcs/shaman.png"
+    profile="portraits/orcs/adept.webp"
+    hitpoints=24
+    movement_type=orcishfoot
+    [defense]
+        village=30
+    [/defense]
+    movement=5
+    experience=50
+    level=1
+    alignment=chaotic
+    advances_to=Orcish Shaman
+    cost=21
+    usage=mixed fighter
+    description= _ "Orcish Shamans are the guardians of orcish magic. Respected among orcish tribes, they form the Orcish Council, which makes important decisions for the whole Orcish community and arbitrates the many conflicts that arise between tribes of this argumentative race. Although physically weak (for orcs), Orcish Shamans are good spell casters and can curse their enemies, draining their life. Adepts are the novice shamans, still young and vigorous, but their spell-casting ability still needs improvement."
+    die_sound={SOUND_LIST:ORC_DIE}
+    {DEFENSE_ANIM "units/orcs/adept-defend-2.png" "units/orcs/adept-defend-1.png" {SOUND_LIST:ORC_HIT} }
+    [attack]
+        name=staff
+        description= _ "staff"
+        type=impact
+        range=melee
+        damage=6
+        number=1
+        icon=attacks/druidstaff.png
+    [/attack]
+    [attack]
+        name=curse
+        description= _ "curse"
+        type=pierce
+        range=ranged
+        damage=7
+        number=2
+        [specials]
+            {WEAPON_SPECIAL_DRAIN}
```

Comment:
so, what do people think about giving the shamans the "Magical" weapon special in addition to Drain? I've given the shamans Magical in SotBE:EE at least; their ranged attack is called "curse" after all, which sounds pretty magical to me... (also I was tired of them missing a lot when testing)

Reply:
The saurians also have curse attack that's magical, for what it's worth.

Reply:
so yeah that seems to be a point in favor of making the curses magical here, too... I'm kind of wondering why it wasn't just made magical in the first place, though? Surely this isn't the first time someone's thinking of it? There must have been *some* sort of reason behind leaving it off, right?

Reply:
> so yeah that seems to be a point in favor of making the curses magical here, too... I'm kind of wondering why it wasn't just made magical in the first place, though? Surely this isn't the first time someone's thinking of it? There must have been _some_ sort of reason behind leaving it off, right?

Well, it was in SotBE and the Shamans might have been like that since it made it necessary to shield them

Reply:
> > so yeah that seems to be a point in favor of making the curses magical here, too... I'm kind of wondering why it wasn't just made magical in the first place, though? Surely this isn't the first time someone's thinking of it? There must have been _some_ sort of reason behind leaving it off, right?
> 
> Well, it was in SotBE and the Shamans might have been like that since it made it necessary to shield them

OK, but if that's the case, then why give them drain, though? Drain is only useful if they get hurt, which can only happen if you're not shielding them... (also, even if you do give them magical, you still have to shield them anyways, since they're still relatively frail...)